buildenv/jenkins/jobs/pull-requests/PullRequest-linux_riscv64

```diff
@@ -0,0 +1,41 @@
+pipeline {
+    agent{label 'Linux && riscv64'}
+    environment {
+        PATH = "/usr/lib/ccache/:$PATH"
+    }
+    stages {
+        stage('Get Sources') {
+            steps {
+                timestamps {
+                    checkout poll: false, scm: [$class: 'GitSCM', branches: [[name: '${sha1}']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[name: 'origin', refspec: '+refs/pull/*:refs/remotes/origin/pr/*', url: 'https://github.com/eclipse/omr.git']]]
+                }
+            }
+        }
+        stage('Build') {
+            steps {
+                timestamps {
+                    echo 'Output CCACHE stats before running and clear them'
+                    sh '''ccache -s -z'''
+
+                    dir('build') {
+                        echo 'Configure...'
+                        sh '''cmake -Wdev -DOMR_COMPILER=ON -DOMR_TEST_COMPILER=ON -DOMR_JITBUILDER=ON -DOMR_JITBUILDER_TEST=ON -C../cmake/caches/Travis.cmake ..'''
```

Comment:
Same comment again, Travis.cmake will enable the various compiler components and testing.